tests/acceptance/cli_test.py

```diff
@@ -1830,6 +1830,20 @@ def test_run_env_values_from_system(self):
         assert 'FOO=bar' in environment
         assert 'BAR=baz' not in environment
 
+    def test_run_label_flag(self):
+        self.base_dir = 'tests/fixtures/run-labels'
+        name = 'service'
+        default = ''
+        foo = 'baz'
+        hello = 'world'
```

Comment:
These 3 variables are only referenced once in the function - maybe we can substitute the value directly?